MAINTAINERS.md

```diff
@@ -59,58 +59,58 @@
 
 ## Release Procedures
 
-When the time comes to release another version of XMonad and Contrib...
+When the time comes to release another version of xmonad and xmonad-contrib:
 
-  1. Create a release branch (e.g., `release-0.XX`).
+  1. Update the version number in all the `*.cabal` files and let the CI
+     verify that it all builds together.
 
-     This will allow you to separate the release process from main
-     development.  Changes you make on this branch will be merged back
-     into `master` as one of the last steps.
+  2. Review documentation files and make sure they are accurate:
 
-  2. Update the version number in the `*.cabal` files and verify
-     dependencies and documentation.  This includes the `tested-with:`
-     field.
+     - [`README.md`](README.md)
+     - [`CHANGES.md`](CHANGES.md)
+     - [`INSTALL.md`](INSTALL.md)
+     - [`man/xmonad.1.markdown.in`](man/xmonad.1.markdown.in)
+     - haddocks <!-- TODO: link to web with doc snapshots once there's one -->
```

Comment:
There is one now! :)

Reply:
Yes, I will squash the fixup before merging :-)

Reply:
Oh!  Didn't realize there was a fixup already, sorry about that